templates/corporate/jobs.html

```diff
@@ -29,28 +29,134 @@ <h1 class="center">Work with us</h1>
+                            users.
+                        </li>
+                        <li>
+                            Design new marketing pages and redesign old ones.
+                            Strengthen our brand's look and feel to make it
+                            instantly recognizable across apps, landing pages,
+                            social accounts, and ads.
+                        </li>
+                        <li>
+                            Prepare icons for web and mobile apps.
+                        </li>
+                    </ul>
+                    <h3>Essential qualifications:</h3>
+                    <ul>
+                        <li>
+                            Your portfolio showcases a track record of designing
+                            high-quality web applications. You had a leading
+                            role in designing a complex application.
+                        </li>
+                        <li>
+                            You have a strong understanding of mobile and web UI
+                            guidelines, standards, approaches, and patterns.
+                            Working within those structures, you can think
+                            creatively to develop elegant solutions to complex
+                            problems.
+                        </li>
+                        <li>
+                            You communicate with clarity and precision, and are
+                            comfortable in a distributed work environment. You
+                            thrive in an inclusive culture where ideas are
+                            debated openly, and everyone is treated with respect.
+                        </li>
+                        <li>Expert knowledge of Figma.</li>
+                    </ul>
+                    <h3>Extra credit for any of the following:</h3>
+                    <ul>
+                        <li>
+                            Familiarity with HTML, CSS and JavaScript.
+                        </li>
+                        <li>
+                            Your portfolio includes mobile applications (iOS or
+                            Android).
+                        </li>
+                        <li>
+                            You've worked on a chat app in the past.
+                        </li>
+                        <li>
+                            You are deeply familiar with a team chat app as an
+                            advanced user or administrator. Better yet, you are an
+                            experienced user of Zulip.
+                        </li>
+                        <li>
+                            You've created an icon set.
+                        </li>
+                    </ul>
+                    <p>
+                        Email us at <a href="mailto:jobs@zulip.com">jobs@zulip.com</a> with your
+                        resume to apply, or learn more about
```

Comment:
Do we not also want a cover letter? I like those as early evidence of communication skills.

Reply:
Oh yeah, I think this is old template text I pulled in. Lemme see what it looks like in more recent ones.